pkg/cli/root.go

```diff
@@ -15,10 +16,16 @@ import (
 	"zotregistry.io/zot/errors"
 	"zotregistry.io/zot/pkg/api"
 	"zotregistry.io/zot/pkg/api/config"
+	extconf "zotregistry.io/zot/pkg/extensions/config"
 	"zotregistry.io/zot/pkg/extensions/monitoring"
 	"zotregistry.io/zot/pkg/storage"
 )
 
+const (
```

Comment:
No need for this. golang "time" package already has a time.Hour, so just make a new const called defaultCVEUpdate = 2*time.Hour